cronjobs/src/commands/purge_old_attachments.py

```diff
@@ -0,0 +1,87 @@
+import os
+from datetime import datetime, timedelta, timezone
+
+from decouple import config
+from google.cloud import storage
+
+from . import KintoClient, fetch_all_changesets
+
+
+SERVER = config("SERVER", default="http://localhost:8888/v1")
+DRY_RUN = config("DRY_RUN", default=False, cast=bool)
+ENVIRONMENT = os.getenv("ENVIRONMENT", "local")
+REALM = os.getenv("REALM", "test")
+STORAGE_BUCKET_NAME = os.getenv(
+    "STORAGE_BUCKET_NAME", f"remote-settings-{REALM}-{ENVIRONMENT}-attachments"
+)
+DELTA_DAYS = config("DELTA_DAYS", default=365, cast=int)
+
+
+def delete_older_files(bucket_name: str, folders: dict[str, datetime]):
+    """
+    Recursively deletes all files older than the given datetime from the specified
+    folders in a GCS bucket.
+
+    @param bucket_name (str): Name of the GCS bucket.
+    @param folders (list[str]): List of folder prefixes (e.g., ['main-workspace/world/', 'security-state/intermediates']).
+    @param cutoff_datetime (datetime): Datetime threshold — files older than this are deleted.
```

Comment:
It shouldn't really matter how old an attachment is. Instead of having a cutoff date, shouldn't we look whether the attachment is still linked from a live record? Attachmetns that are still linked from live records are still needed, even if they are old. Attachments that aren't linked from live records can be deleted.